test/test_autograd.py

```diff
@@ -7765,6 +7764,24 @@ def test_out_variant(self):
             with self.assertRaisesRegex(RuntimeError, "out= function"):
                 torch.add(foo, bar, out=bar)
 
+    # TODO: Temporary solution to substitute for OpsInfo
+    def test_mkldnn_convolution(self):
+        inp = torch.rand(2, 2, 5, 5, dtype=torch.float, requires_grad=True)
+        weight = torch.rand(2, 2, 3, 3, dtype=torch.float, requires_grad=True)
+        bias = torch.rand(2, dtype=torch.float, requires_grad=True)
+
+        padding = (2, 3)
```

Comment:
I'm still not sure that padding is very representative. For a 3x3 kernel, people usually use (1, 1) no?